developer-model-service/docs/models.md

```diff
@@ -0,0 +1,13 @@
+# Supported Models
+
+The following models are accessible via this model server API.
+
+## IBM Granite-8B-Code-Instruct
+
+[Model Repository](https://huggingface.co/ibm-granite/granite-8b-code-instruct)
+
+<!--
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+
+[Model Documentation](../../../../default/resource/ibm-granite-8b-code-instruct)
```

Comment:
This differs from what is in the `main` branch of the catalog https://github.com/redhat-ai-dev/model-catalog/blob/main/multi-model-service/docs/models.md?plain=1#L11 and the link contains `default/default` and can't find the docs. I know it's super finicky since we basically have to `../` to route the url

Reply:
I found that the link to the model resource docs wouldn't work, depending on where you navigated from. If you had accessed from the `Docs` tab on the left, the URL would be slightly different

That being said, I found that absolute paths did work for both, so I'll switch to `/docs/default/resource/ibm-granite-8b-code-instruct` (for example)